packages/commonwealth/server/routes/status.ts

```diff
@@ -30,6 +30,7 @@ type StatusResp = {
   roles?: RoleInstanceWithPermission[];
   loggedIn?: boolean;
   user?: {
+    id: number;
```

Comment:
Per Knock instructions, we should enable the enhanced security mode and return a Knock JWT. I'm fine with getting this PR in as is, as long as we link the follow-up security ticket before merging so that it is not forgotten.

https://docs.knock.app/in-app-ui/security-and-authentication#authentication-with-enhanced-security

Reply:
Good catch. I think we should fix the security issues first.  They shouldn't allow this mode of the SDK actually.  It's too easy to spoof another user otherwise.

This isn't a complicated fix so I will do this next.